workflows/amplicon/qiime2/qiime2-I-import/.dockstore.yml

```diff
@@ -0,0 +1,47 @@
+version: 1.2
+workflows:
+- name: QIIME2-Ia import multiplexed data single end
+  subclass: Galaxy
+  publish: true
+  primaryDescriptorPath: /QIIME2-Ia-multiplexed-data-single-end.ga
+  testParameterFiles:
+  - /QIIME2-Ia-multiplexed-data-single-end-tests.yml
+  authors:
+  - name: Debjyoti Ghosh
+    orcid: 0009-0008-1496-1677
+  - name: "Helmholtz-Zentrum f\xFCr Umweltforschung - UFZ"
+    address: "Permoserstra\xDFe 15, 04318 Leipzig"
```

Comment:
That's interesting, can you fix the umlaut here ? Seems like a bug we should fix.